### DISCUSSION TITLE
proxxy

### DISCUSSION THREAD

**Discussion by pl4udible** [3kh0/ext-remover#535] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
free proxxys please all of mines r  caught by the school online and blocked

**Comment by obkduishinijf** [comment, marked as answer]
if you can make a github account and post then you can do a bit of googling, lazy

**Comment by pl4udible** [comment]
their all blocked by the school as its classified as security

**Comment by pl4udible** [comment]
heroku app works

**Reply by rwg100** [reply]
you have to pay for that

**Comment by rwg100** [comment]
[https://bananaboat.me/ <](url)
[https://browser.rammerhead.org/](url)

**Comment by pl4udible** [comment]
dont work

i

**Reply by qDread-dev** [reply]
> dont work
> 
> i

reply to what people say and not the entire post 💀

**Reply by obkduishinijf** [reply]
> > dont work
> > i
> 
> reply to what people say and not the entire post 💀

fr 💀

**Comment by amelia-bartlett** [comment]
https://first-neon-honeycup.glitch.me/ (cannot vouch for its integrity, only its existence as of posting)

**Reply by GGboogie** [reply]
It doesn't work anymore btw, tried it at: 4-20-23 10:32 AM.
